pennylane/vqe/vqe.py

```diff
@@ -30,6 +30,33 @@
 OBS_MAP = {"PauliX": "X", "PauliY": "Y", "PauliZ": "Z", "Hadamard": "H", "Identity": "I"}
 
 
+def _compute_grouping_indices(observables, grouping_type="qwc", method="rlf"):
+
+    # todo: directly compute the
+    # indices, instead of extracting groups of observables first
+    observable_groups = qml.grouping.group_observables(observables, coefficients=None, grouping_type=grouping_type, method=method)
+
+    observables = copy(observables)
+
```

Comment:
The below is basically just copied from the existing `group_observables` transform when coefficients are given...